composer.json

```diff
@@ -1,20 +1,24 @@
 {
     "name": "laminas/laminas-mvc-plugins",
-    "description": "Collection of all stable laminas-mvc plugins served via separate packages.",
     "type": "metapackage",
-    "license": "BSD-3-Clause",
+    "description": "Collection of all stable laminas-mvc plugins served via separate packages.",
     "keywords": [
         "laminas",
         "mvc",
         "plugins"
     ],
     "homepage": "https://laminas.dev",
-    "support": {
-        "issues": "https://github.com/laminas/laminas-mvc-plugins/issues",
-        "source": "https://github.com/laminas/laminas-mvc-plugins",
-        "rss": "https://github.com/laminas/laminas-mvc-plugins/releases.atom",
-        "chat": "https://laminas.dev/chat",
-        "forum": "https://discourse.laminas.dev"
+    "license": "BSD-3-Clause",
+    "require": {
+        "php": "^7.3 || ~8.0.0",
```

Comment:
Can you check if we can drop the PHP version constraint? The underlying packages already define the requirements for PHP. Thanks in advance!

Reply:
Sure, that will work.